source/documentation/administration_guide/topics/Understanding_Storage.adoc

```diff
@@ -5,6 +5,10 @@
 A storage domain is a collection of images that have a common storage interface. A storage domain contains complete images of templates and virtual machines (including snapshots), or ISO files. A storage domain can be made of block devices (SAN - iSCSI or FCP) or a file system (NAS - NFS, GlusterFS, or other POSIX compliant file systems).
 
 By default, GlusterFS domains and local storage domains support 4K block size. 4K block size can provide better performance, especially when using large files, and it is also necessary when you use tools that require 4K compatibility, such as VDO.
+[NOTE]
```

Comment:
Please mark this only for RHV. oVirt will keep consuming gluster as long as gluster will ship updates.

Reply:
Same in the other sections with deprecation notice